docker/storage/mysql/Dockerfile

```diff
@@ -16,13 +16,11 @@ FROM alpine
 LABEL MAINTAINER Zipkin "https://zipkin.io/"
 
 WORKDIR /mysql
-ADD docker/storage/mysql/install.sh /mysql/install
-ADD docker/storage/mysql/configure.sh /mysql/configure
+ADD docker/storage/mysql/install.sh /mysql/install.sh
```

Comment:
Bonus points might be `ADD docker/storage/mysql/*.sh /mysql/`

Reply:
oh .. didn't know that worked